### PR TITLE
VoodooPS2 and other clarifications

### PR DESCRIPTION
**Pull request by krisek**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -279,6 +285,10 @@ SSDT-SBUS-MCHC.aml
 
 >**Note**: `SSDT-HPET` and it's `config.plist` patch is only required for eMMC support.
 
+>**Note**: Don't try to shortcut by using precompiled `SSDT-EC.aml` & `SSDT-USBX.aml`, the ones generated by `SSDTTime` should be the best.
+
+>**Note**: For additional power management you can add `SSDT-PLUG-DRTNIA`.
```
Comment (meghan06) [inline README.md:290]: This is not needed.

File: README.md [hunk 2]
```diff
@@ -256,7 +256,12 @@ VoodooPS2Controller.kext
 WhateverGreen.kext
 ```
 
->**Note**: If you plan to install macOS Big Sur or older (11≥), your Bluetooth kexts will be different. Read the Dortania guide to find out what you need.
+>**Note**: 
+> - If you plan to install macOS Big Sur or older (11≥), your Bluetooth kexts will be different. Read the Dortania guide to find out what you need.
+> - You might need to clean-up PlugIns in `VoodooPS2Controller.kext` except of `VoodooPS2Keyboard.kext`. `VoodooI2C.kext` and `VoodooI2CELAN.kext`from separate repo are still needed if you go down this path.
+> - Don't add blindly `IntelBluetoothInjector.kext` from the `IntelBluetoothFirmware` repo, it conflicts with `EmeraldSDHC` and external USB.
+> - Most of the USB seems to work without USB mapping nowadays (tested with C433), hence `UTBMap.kext` & `USBToolBox.kext` are optional.
```
Comment (meghan06) [inline README.md:263]: Correct, but it is good practice to do so.

File: README.md [hunk 3]
```diff
@@ -256,7 +256,12 @@ VoodooPS2Controller.kext
 WhateverGreen.kext
 ```
 
->**Note**: If you plan to install macOS Big Sur or older (11≥), your Bluetooth kexts will be different. Read the Dortania guide to find out what you need.
+>**Note**: 
+> - If you plan to install macOS Big Sur or older (11≥), your Bluetooth kexts will be different. Read the Dortania guide to find out what you need.
+> - You might need to clean-up PlugIns in `VoodooPS2Controller.kext` except of `VoodooPS2Keyboard.kext`. `VoodooI2C.kext` and `VoodooI2CELAN.kext`from separate repo are still needed if you go down this path.
+> - Don't add blindly `IntelBluetoothInjector.kext` from the `IntelBluetoothFirmware` repo, it conflicts with `EmeraldSDHC` and external USB.
```
Comment (meghan06) [inline README.md:262]: It should not conflict, it's fine on my C425.

File: README.md [hunk 4]
```diff
@@ -256,7 +256,12 @@ VoodooPS2Controller.kext
 WhateverGreen.kext
 ```
 
->**Note**: If you plan to install macOS Big Sur or older (11≥), your Bluetooth kexts will be different. Read the Dortania guide to find out what you need.
+>**Note**: 
+> - If you plan to install macOS Big Sur or older (11≥), your Bluetooth kexts will be different. Read the Dortania guide to find out what you need.
+> - You might need to clean-up PlugIns in `VoodooPS2Controller.kext` except of `VoodooPS2Keyboard.kext`. `VoodooI2C.kext` and `VoodooI2CELAN.kext`from separate repo are still needed if you go down this path.
```
Comment (meghan06) [inline README.md:261]: This shouldn't be needed, let me check anyway.

**Review by meghan06 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (krisek) [reply README.md:290]: 👍

File: README.md (hunk 2 above)
Comment (krisek) [reply README.md:263]: 👍

**Comment by meghan06** [comment]
Your changes have been added into my rewrite, see https://github.com/meghan06/ChromebookOSX/pull/21.

**Pull request closed without merging**